lib/models/MatrixAction.js

```diff
@@ -45,7 +45,13 @@ MatrixAction.fromEvent = function(client, event) {
                 fileSize = " (" + Math.round(event.content.info.size / 1024) +
                     "KB)";
             }
-            text = client.mxcUrlToHttp(event.content.url) +
+
+            // By default assume that the media server = client homeserver
+            if (!mediaUrl) {
+                mediaUrl = client.getHomeserverUrl();
+            }
+
+            text = client.mxcUrlToAlternateHttp(mediaUrl, event.content.url) +
```

Comment:
Done, but how shall we include https://github.com/matrix-org/matrix-appservice-bridge/pull/31 if mx-as-irc is pinned at v1.3 of mx-as-bridge?